trivup/trivup.py

```diff
@@ -283,6 +284,8 @@ def __init__(self, name):
         self.name = name
         if name == 'localhost':
             self.exec_cmd = ''
+        elif name == 'docker':
+            self.exec_cmd = 'docker'
```

Comment:
This is not really the same thing, should probably be `docker run ..` if anything.

Reply:
thats fair maybe it just doesn't belong here at all.